src/systemTest/targets/soot/interfaceTesting/TestDerivedInterfaces.java

```diff
@@ -13,7 +13,7 @@ default void print() {
 	}
 }
 
-interface interfaceTestTwo extends interfaceOne{
+interface interfaceTestTwo extends interfaceTestOne{
```

Comment:
In Java classes and Interfaces start with a capital letter.

Reply:
Fixed it